bin/metrics-docker-stats.rb

```diff
@@ -88,8 +80,8 @@ class DockerStatsMetrics < Sensu::Plugin::Metric::CLI::Graphite
          default: false
 
   option :name_parts,
-         description: 'Partial names by spliting and returning at index(es). \
-                       eg. -m 3,4 my-docker-container-process_name-b2ffdab8f1aceae85300 for process_name.b2ffdab8f1aceae85300',
+         description: 'Partial names by spliting and returning at index(es).
+         eg. -m 3,4 my-docker-container-process_name-b2ffdab8f1aceae85300 for process_name.b2ffdab8f1aceae85300',
```

Comment:
wont this add a bunch of weird whitespace to the description?

Reply:
The original display : 
````
    -m, --match index                Partial names by spliting and returning at index(es). \
                       eg. -m 3,4 my-docker-container-process_name-b2ffdab8f1aceae85300 for process_name.b2ffdab8f1aceae85300
````

The updated one : 
````
    -m, --match index                Partial names by spliting and returning at index(es).
         eg. -m 3,4 my-docker-container-process_name-b2ffdab8f1aceae85300 for process_name.b2ffdab8f1aceae85300
````

For better code readability, I think it is required to align in multi-lines description , the first word on the second line with the single quote on the first line indeed.

I had looked to an easy solution to align multiple line description with mixlib inside code and with --help rendering but didn't find anything.

I prefer a render like this : 
````
    -m, --match index                Partial names by spliting and returning at index(es).
                                     eg. -m 3,4 my-docker-container-process_name-b2ffdab8f1aceae85300 for process_name.b2ffdab8f1aceae85300
````
But it will required a code like this.... : 
````
         description: 'Partial names by spliting and returning at index(es).
                                      eg. -m 3,4 my-docker-container-process_name-b2ffdab8f1aceae85300 for process_name.b2ffdab8f1aceae85300',
````

Reply:
makes sense